launch_testing/launch_testing/markers.py

```diff
@@ -45,6 +51,7 @@ def _wrapper(self, *args, **kwargs):
                     self._outcome.errors.clear()
                     self._outcome.success = True
                     n -= 1
+                time.sleep(delay)
```

Comment:
A sleep of `0` might still change behavior since it might suspend the calling thread (not sure how that is implemented in Python).

Reply:
True, if you think it's better we can default to `None` and avoid sleeping altogether in this case.

Reply:
See 1c4be5f